lib/ember-test-helpers/isolated-container.js

```diff
@@ -83,5 +84,8 @@ export default function isolatedContainer(fullNames) {
     var normalizedFullName = resolver.normalize(fullName);
     container.register(fullName, resolver.resolve(normalizedFullName));
   }
-  return container;
+  return {
+    container: container,
+    registry: registry
+  };
 }
```

Comment:
I can just implement `isolatedRegistry` instead. `isolatedContainer` can keep the same public API, but get reimplemented in terms of `isolatedRegistry`.

(While also maintaining compatibility for old Embers that don't have a registry.)